src/deadline/maya_submitter/job_bundle_output_test_runner.py

```diff
@@ -136,17 +136,7 @@ def run_maya_render_submitter_job_bundle_output_test():
         gui_error_handler("Error running job bundle output test", mainwin),
         _consistent_machine_settings(),
     ):
-
-        default_tests_dir = Path(__file__).parent
-        while (
-            len(default_tests_dir.parts) > 1
-            and not (default_tests_dir / "job_bundle_output_tests").is_dir()
-        ):
-            default_tests_dir = default_tests_dir.parent
-        if len(default_tests_dir.parts) == 1:
-            default_tests_dir = Path(__file__).parent
-        else:
-            default_tests_dir = default_tests_dir / "job_bundle_output_tests"
+        default_tests_dir = Path(__file__).parent.parent.parent.parent / "job_bundle_output_tests"
```

Comment:
Did you mean to remove this and the message below about skipping the test when Renderman for Maya isn't installed? Maybe a rebase/merge thing here.

Reply:
This is my first time trying it, but because you had the "maintainers can edit your PR" option checked, I was able to use the github code editing to do this myself. :)